docs/source/tutorial/cli.rst

```diff
@@ -12,6 +12,7 @@ Command-Line Interface
     studies, Show a list of studies.
     study optimize, Start optimization of a study.
     study set-user-attr, Set a user attribute to a study.
+    storage upgrade, Upgrade the schema of a storage to up-to-date.
```

Comment:
Please sort the entries alphabetically.

[nits] This may be my personal preference, but `to up-to-date` seems redundant because the word upgrade implies it.

Reply:
Thank you for your comment. I will modify the order and remove the redundant words.

Reply:
Fixed: https://github.com/pfnet/optuna/pull/336/commits/fec4de4bcf7a8c6df544bdb4200c380fc1210474